### PR TITLE
ping-pong-game

### PR DESCRIPTION
**Pull request by mayank-del**
Description
This this basically a game which contains python turtle graphics

Fixes https://github.com/Kushal997-das/Project-Guidance/issues/329

Checklist
[ y] I've read the contribution guidelines.
[ y] I've checked the issue list before deciding what to submit.
[y ] I've edited the README.md and link to my code.
Related Issues or Pull Requests
NO issues , it is just a feature , I want to add.

### CODE REVIEW

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@mayank-del Your folder not in correct place , You folder location should be like this DesktopApplication/Python/Basic/Project_name.

**Comment by mayank-del** [comment]
Sir, now i did whatever you told , i added it inside python of desktop application, now ,please accept it.

File: Desktop Application/Basic/Python/Ping-pong-Python/Readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+###### this ping pong game is based on turtle graphics library.
+
+###### How to run the game? 
+
+##  Just open any code editor and go to main.py and click on run run button of your code editor, it will run succesfully
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Ping-pong-Python/Readme.md:5]: Frame the sentence correctly .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@mayank-del Add gif of your project output in `readme.md` file . Make sure you do the changes soon , else we gonna close this PR as it's been more than 4days you were not responding , Do the changes by the EOD .

**Pull request closed without merging**